src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -102,7 +108,9 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
                 var returnTypeName = ztSearchElement.Descriptor.ReturnType.Name;
 
                 var descriptor = ztSearchElement.Descriptor;
-                if ((returnTypeName == inputPortType)
+                if (descriptor.ReturnType.ToString() == inputPortType)
+                    if (!skippedInputPorts.Any(s => s == returnTypeName.ToString()) 
```

Comment:
@zeusongit @mjkkirschner Seems @aparajit-pratap  The goal here is to let `elements` remain empty for these specific inputPortTypes. Should we move the check out side of the scanning to save performance? e.g. moving to line 96 since we know the results are not going to be useful?

Reply:
seems like a good idea, will be updating